src/app/business/block/buckets.service.ts

```diff
@@ -103,7 +103,32 @@ export class BucketService {
     let url = this.url + "/" +name;
     return this.http.delete(url,param);
   }
-
+  //copy object
+  copyObject(object,param, options){
```

Comment:
Please remove copyObject code from this PR. Keep it in the PR for copyObject

Reply:
@anvithks I have removed copyObject code, please check, thank you